public/sum.js

```diff
@@ -1,4 +1,4 @@
 function sum(a, b) {
```

Comment:
This file was for testing? Do you still need it?

Reply:
Meant to remove or gitignore it, Ram pointed it out as well. Removed the file.